vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -869,6 +869,18 @@ testBccTools () {
   return 0
 }
 
+testNBCParserBinary () {
+  local test="testNBCParserBinary"
+  local go_binary_path="/opt/azure/containers/nbcparser"
+
+  echo "$test: checking existence of nbcparser go binary at $go_binary_path"
+  if [ ! -f "$go_binary_path" ]; then
+    err "$test: nbcparser go binary does not exist at $go_binary_path"
+    return 1
+  fi
+  echo "$test: nbcparser go binary exists at $go_binary_path"
```

Comment:
should we also invoke it and assert exit 0?

basically just call the help command like `nbcparser -h` and make sure it works

Reply:
yea i was thinking about this, but i need a sample json with some test data in order to trigger it, so not sure if its work adding that to the vhd as well

Reply:
we should, if its wrong compiled, it won't be of any use, we should ensure that it's working and ready to be executed.